dev-util/gperf/gperf-3.1-r2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
We don't need a new revision for this: https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html